res/com.github.alexhuntley.Plots.metainfo.xml

```diff
@@ -5,10 +5,12 @@
   <metadata_license>FSFAP</metadata_license>
   <project_license>GPL-3.0-or-later</project_license>
   <name>Plots</name>
+  <name xml:lang="es">Plots</name>
   <name xml:lang="fr">Plots</name>
   <name xml:lang="hr">Grafikoni</name>
   <developer_name>Alex Huntley</developer_name>
   <summary>Simple graph plotting</summary>
+  <summary xml:lang="es">Trazado de funciones sencillas</summary>
```

Comment:
I don't know much Spanish, but shouldn't it be "sencillo" (singular)? It is the app which should be simple, not (necessarily) the functions being plotted.

Reply:
It should in that case 😅 It's fixed now.